package.json

```diff
@@ -50,8 +50,8 @@
     "preact": "^7.2.1",
     "preact-compat": "^3.17.0",
     "prop-types": "^15.6.0",
-    "react": "^15.6.2",
-    "react-dom": "^15.6.2"
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0"
```

Comment:
Should `peerDependencies ` be updated to ^16.x only? 🤔

Reply:
Great catch